assets/i18n/ar.json

```diff
@@ -351,6 +351,7 @@
 	"I18N_PLAYER_LOADING": "جارٍ التحميل...",
 	"I18N_PLAYER_NEXT_LESSON": "الدرس التالي",
 	"I18N_PLAYER_NO_OBJECTIVE": "لا يوجد هدف محدد.",
+	"I18N_PLAYER_IS_PRIVATE": "الاستكشاف خاصة.",
```

Comment:
I think you've got this in the wrong place, here and elsewhere. Make sure to alphabetize by key or your PR will fail the Travis tests.

Reply:
Okay! Now this will be tiring!